commands/game/suicide.js

```diff
@@ -21,12 +22,29 @@ module.exports = {
             }
         } else if (message.channel.name.includes("priv") || message.channel.name == "day-chat") {
             if (!message.member.roles.cache.has(ids.alive)) return
-            db.set(`suicided_${message.author.id}`, true)
-            let day = message.guild.channels.cache.find((c) => c.name === "day-chat")
-            let role = await db.fetch(`role_${message.author.id}`)
-            day.send("**" + message.member.nickname + " " + message.author.username + " (" + role + ")** has commited suicide!")
-            message.member.roles.add(ids.dead)
-            message.member.roles.remove(ids.alive)
+            let row = new MessageActionRow({ components: [new MessageButton({ style: "SUCCESS", label: "Suicide", customId: "suicide" }), new MessageButton({ style: "DANGER", label: "Cancel", customId: "cancel" })] })
+            message.channel.send({ embeds: [new MessageEmbed({ color: "DARK_ORANGE", title: "Are you sure you want to commit suicide?", description: "You will get a soft warn if you commit suicide now." })], components: [row] }).then((msg) => {
```

Comment:
We call them game warns now

Reply:
Tbh people will be confused with this game warn and gamewarn ~~softwarn~~ which will be used soon